modules/navier_stokes/examples/solidification/galium_melting-physics.i

```diff
@@ -0,0 +1,225 @@
+    ny = ${Ny}
+  []
+[]
+
+[AuxVariables]
+  [fl]
+    type = MooseVariableFVReal
+    initial_condition = 0.0
+  []
+[]
+
+[AuxKernels]
+  # Note that because this is only executed on timestep end,
+  # the phase fractions are constant over each time step
+  [compute_fl]
+    type = NSLiquidFractionAux
+    variable = fl
+    temperature = T_fluid
+    T_liquidus = '${T_liquidus}'
+    T_solidus = '${T_solidus}'
+    execute_on = 'TIMESTEP_END'
+  []
+[]
+
+[Physics]
+  [NavierStokes]
+    [Flow]
+      [flow]
+        compressibility = 'incompressible'
+
+        pin_pressure = true
+        pinned_pressure_type = 'average'
+        pinned_pressure_value = 0
+
+        velocity_variable = 'vel_x vel_y'
+
+        density = 'rho_mixture'
+        dynamic_viscosity = ${mu}
+
+        # Initial conditions
+        initial_velocity = '0 0 0'
+        initial_pressure = 0
+
+        # Boundary conditions
+        wall_boundaries = 'left right top bottom'
+        momentum_wall_types = 'noslip noslip noslip noslip'
+
+        # Friction
+        friction_types = "Darcy Forchheimer"
+        friction_coeffs = "Darcy_coefficient Forchheimer_coefficient"
+
+        # Boussinesq
+        boussinesq_approximation = true
+        gravity = '0 -9.81 0'
+        ref_temperature = ${T_cold}
+        thermal_expansion = 'alpha_b'
+        # Solid phase is not moving
+        density_gravity = ${rho_liquid}
+
+        mass_advection_interpolation = '${advected_interp_method}'
+        momentum_advection_interpolation = '${advected_interp_method}'
+        velocity_interpolation = '${velocity_interp_method}'
+      []
+    []
+    [FluidHeatTransfer]
+      [energy]
+        coupled_flow_physics = flow
+
+        thermal_conductivity = 'k_mixture'
+        specific_heat = 'cp_mixture'
+
+        initial_temperature = '${T_cold}'
+
+        # See flow physics for wall names
+        energy_wall_types = 'fixed-temperature fixed-temperature heatflux heatflux'
+        energy_wall_functors = '${T_hot} ${T_cold} 0 0'
+
+        energy_advection_interpolation = '${advected_interp_method}'
+      []
+    []
+    [TwoPhaseMixture]
+      [mixture]
+        add_phase_transport_equation = false
+        liquid_phase_fraction_name = 'phase_1'
+        dispersed_phase_fraction_name = 'fl'
+
+        fluid_heat_transfer_physics = energy
+        add_phase_change_energy_term = true
+        use_dispersed_phase_drag_model = false
+
+        # Base phase material properties
+        first_phase_density_name = ${rho_liquid}
+        first_phase_viscosity_name = ${mu}
+        first_phase_specific_heat_name = ${cp_liquid}
+        first_phase_thermal_conductivity_name = ${k_liquid}
+        output_all_properties = true
+
+        # Other phase material properties
+        other_phase_density_name = ${rho_solid}
+        other_phase_viscosity_name = ${mu}
+        other_phase_specific_heat_name = ${cp_solid}
+        other_phase_thermal_conductivity_name = ${k_solid}
+      []
```

Comment:
So I would vote to continue forward with what we have already for consistency. I would vote for `phase_1` and `phase_2`. I got really confused because `dispersed_phase_fraction_name` is pointing to the fluid fraction for which you have the dedicated parameter `liquid_phase_fraction_name` (which doesn't point to the liquid fraction??). And then later within the same parameter block we switch from a liquid-dispersed naming paradigm to first-other

Reply:
renamed. i did this to get around the "phase 1 is an auxvariable now" and yes it was confusing. 

Works now imo